src/app/core/ui/search/search.component.html

```diff
@@ -1,71 +1,91 @@
-<div class="white mat-subtitle-2 remove-margin-bottom full-width">
-  <mat-form-field class="full-width searchbar">
+<div class="white mat-subtitle-2 remove-margin-bottom">
+  <button
+    *ngIf="mobile && !searchActive"
```

Comment:
could use `@if(mobile && !searchActive) { ... } @else {...}` to make the link between the two conditions slightly more obvious